APE_finance.rst

```diff
@@ -0,0 +1,143 @@
+1. Pay for things that are important and otherwise won’t get done. *Rationale: there is much more work to be done than there are funds to do all of that work. There are also many volunteers and externally sponsored people interested and willing to do some of the work without compensation from the Project.*
+2. Don’t rely on funding always being there. Plan for sustainability: how will activities be continued when the funds dry up? Invest in documentation, infrastructure, and easy to maintain workflows. For example, funded work on governance should always record the process clearly so that volunteers who follow have less work to do to figure it out. 
+3. Consider potential positive benefits to Astropy maintainers and contributors, maintainers of other projects, end users, and other stakeholders like packagers and educators. 
+4. Also consider the long-term costs to Astropy maintainers and contributors, to counterbalance 3.
+5. Do not increase the load on volunteer maintainers by assuming they are available to review or otherwise support the work. Instead, proposals should include paid time for reviews and other necessary activities. *Rationale: the effect of spending funds should be positive for everyone, not just for the people getting paid. This is also a matter of fairness.*
+
+Restricted / earmarked funding sources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Much of the Project’s funding comes from specific grants from private foundations and government agencies. In these cases, the fundable activities are explicitly described in the proposal text. Such funds must be spent following the proposed plan and often come with further restrictions on who can be funded. For example, NASA funding can only go to individuals or institutions in the USA and EU funding is generally restricted to institutions or individuals in the EU or associated countries. The Finance Committee should make these restrictions clear to the community so that they may submit funding requests aligned with the existing grants.
+
+Whom to pay
+^^^^^^^^^^^
+1. All else being equal, give preference to existing maintainers/contributors.
+2. When looking outside of the current team, consider this an opportunity to make the project more diverse.
+3. Pay attention to the following when considering paying someone:
+    * the necessary skills to execute the tasks,
+    * communication and self-management skills,
+    * experience contributing to and working with open source projects,
+    * likelihood of long-term involvement with the Astropy community.
+
+It will likely depend on the particular activity as to whether there’s already a clear best candidate within the Astropy team, or if we must look for new people to get involved. This decision should be taken in accordance with documented Project governance procedures. If the position is not filled from within the project, it should generally be publicly advertised to give a broader group of people a chance to apply for it.
```

Comment:
```suggestion
It will likely depend on the particular activity as to whether there’s already a clear best candidate within the Astropy contributor base, or if we must look for new people to get involved. This decision should be taken in accordance with documented Project governance procedures. If the position is not filled from within the project, it should generally be publicly advertised to give a broader group of people a chance to apply for it.
```